virt/vm_templates/virt-deleting-vm-template.adoc

```diff
@@ -4,8 +4,6 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-deleting-vm-template
 toc::[]
 
-You can delete a virtual machine template in the web console.
+Using the web console, you can delete any virtual machine template that was created from a Red Hat template or by using the interactive wizard. Red Hat virtual machine templates cannot be deleted.
```

Comment:
This particular module focuses on deleting virtual machine templates. Reorganized the paragraph per your guidance.

Regarding your question on the creation of virtual machine templates (which is in a different module):
You can select a Red Hat template and create a virtual machine template that is prefilled with the settings available in the preconfigured Red Hat templates:

https://cnv-9030-vm-templates--ocpdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-creating-vm-template.html#virt-understanding-vm-templates-web_virt-creating-vm-template

Would it help if I expanded the topic with this info:

When you select a preconfigured Red Hat template, the interactive wizard prompts you to add a boot source if a boot source is unavailable for that template. Then, you can save your custom template that is based on the settings provided in the Red Hat template. Using a Red Hat template is convenient because each template is preconfigured with the settings for the operating system, flavor, workload type and other required details. 

If you select the interactive wizard to create a virtual machine template, the wizard prompts you to create a custom template where you provide the configuration details for the operating system, flavor, workload type and other settings.